goagen/gen_client/cli_generator.go

```diff
@@ -693,4 +732,99 @@ func hasFlag(name string) bool {
 	}
 	return false
 }
-`
+
+func jsonVal(val string) *interface{} {
+	var t interface{}
+	err := json.Unmarshal([]byte(val), t)
+	if err != nil {
+		panic(err)
```

Comment:
So, bubble it all the way up to Run() ? Convert each array/type individually in Run, for so to log and return err out of run?